pkg/ocserv/ocserv.html

```diff
@@ -0,0 +1,173 @@
+			id="status_table">
+			</table>
+		</div>
+  	</div>
+
+	<div class="panel panel-default">
+		<div class="panel-heading">
+			<span id="head">Connected users</span>
+		</div>
+		<table class="table table-hover" style="margin-bottom:10px"
+		id="users_table">
+	        <thead>
+	        <tr>
+	          <th class="users-id">ID</th>
+  	          <th class="users-name">Username</th>
+	          <th class="users-remote-ip">Remote IP</th>
+	          <th class="users-ip">VPN IP</th>
+		  <th class="users-since">Since</th>
+		  <th class="users-state">State</th>
+		</tr>
+	        </thead>
+	        <tbody id="users_table_tbody">
+
+	        </tbody>
+		</table>
+	</div>
+    </div>
+
+
+<script>
+require([
+    'jquery',
+    'base1/cockpit'
+], function($, cockpit) {
+        var raw;
+        var status_raw;
+
+        occtl_users_run();
+        occtl_status_run();
+
+        function occtl_users_run() {
+            var proc = cockpit.spawn(["/usr/bin/occtl", "-n", "-j", "show", "users"]);
+            proc.done(occtl_users_success);
+            proc.stream(occtl_users_output);
+            proc.fail(occtl_users_fail);
+
+            raw = "";
+        }
+
+        function occtl_users_success() {
+            var obj;
+            try {
+            	obj = JSON.parse(raw);
+	    } catch(e) {
+	    	console.warn(e, obj);
+	    }
+
+	    $("#users_table_tbody").empty();
+
+	    $.each(obj, function (index,Object) {
+	    	var ipv4 = Object.IPv4;
+	    	var ipv6 = Object.IPv6;
+	    	var ip;
+
+	    	if (ipv4 && ipv6) {
+		    ip = ipv4 + "/" + ipv6;
+		} else {
+		    if (ipv4)
+			ip = ipv4;
+		    else
+			ip = ipv6;
+		}
+
+	    	$("#users_table").append('<tr><td class="users-id">' + Object.ID + '</td>' +
+		    '<td class="users-name">' + Object.Username + '</td>' + 
+	    	    '<td class="users-remote-ip">' + Object["Remote IP"] + '</td>' + 
+	    	    '<td class="users-ip">' + ip + '</td>' +
+	    	    '<td class="users-since">' + Object["Connected at"] + '</td>' +
+	    	    '<td class="users-state">' + Object.State + '</td>' +
+		    '</tr>').toString();
+	    });
+        }
+
+        function occtl_users_fail() {
+        }
+
+        function occtl_users_output(data) {
+            raw = raw.concat(data);
+        }
+
+        function occtl_status_run() {
+            var proc = cockpit.spawn(["/usr/bin/occtl", "-n", "-j", "show", "status"]);
+            proc.done(occtl_status_success);
+            proc.stream(occtl_status_output);
+            proc.fail(occtl_status_fail);
+
+            status_raw = "";
+        }
+
+        function occtl_status_success() {
+            var obj;
+            try {
+            	obj = JSON.parse(status_raw);
+	    } catch(e) {
+	    	console.warn(e, obj);
+	    }
+
+	    $("#status_table").empty();
+
+	    if (!obj) {
+	        $("#status_table").append('<tr><td><td>Status</td>offline</td></tr>').toString();
+	    } else {
+		$("#status_table").append('<tr><td>Status</td><td>' +
```

Comment:
for increased readability this would be an ideal candidate to use Mustache templates, an example is [in storaged](https://github.com/cockpit-project/cockpit/blob/master/pkg/storaged/devices.html#L46). With the variables in the template named accordingly, `obj` could probably be passed directly to the Mustache renderer.